front-end/src/components/Dashboard/index.js

```diff
@@ -169,8 +183,13 @@ const DashboardUserContent = styled.div`
 `;
 
 const EditProfileLinkContainer = styled.div`
-  display: flex;
-  font-size: 12px;
+   display: flex;
+   flex-direction: column;
+   padding: 25px 25px 25px 25px;
```

Comment:
If all sides are the same, you can just put padding: 25px

Reply:
fixed